tensorboard/webapp/widgets/data_table/data_table_component.ts

```diff
@@ -200,4 +211,50 @@ export class DataTableComponent {
     }
     this.sortDataBy.emit({header, order: SortingOrder.ASCENDING});
   }
+
+  dragStart(header: ColumnHeaders) {
+    this.draggingHeader = header;
+
+    // This stop the end drag animation
+    document.addEventListener('dragover', function (e) {
+      e.preventDefault();
+    });
```

Comment:
It would be nice to remove this listener when either:
* The drag stops
* The component is destroyed

Reply:
Done.